SingularityService/src/main/java/com/hubspot/singularity/SingularityService.java

```diff
@@ -78,7 +82,7 @@ public void initialize(final Bootstrap<T> bootstrap) {
 
     guiceBundle = GuiceBundle.defaultBuilder(SingularityConfiguration.class)
         .modules(
-            new SingularityServiceModule(),
+            getServiceModule(),
             getObjectMapperModule(),
             new SingularityAuthModule()
```

Comment:
might want to add a get method for this module as well, for consistency.

Reply:
👍 I have some groups/scopes work coming up soon, will likely do that in a future PR